plinth/modules/dynamicdns/templates/dynamicdns_configure.html

```diff
@@ -0,0 +1,165 @@
+          $('#id_dynamicdns-dynamicdns_update_url').closest('.form-group').hide();
+          $('#id_dynamicdns-disable_SSL_cert_check').closest('.form-group').hide();
+          $('#id_dynamicdns-use_http_basic_auth').closest('.form-group').hide();
+          $('#id_dynamicdns-dynamicdns_server').closest('.form-group').show();
+        }
+      }
+
+      function configure_dropdown()
+      {
+        if ($("#id_dynamicdns-service_type option:selected").text() == "GnuDIP") {
+          $('#id_dynamicdns-dynamicdns_update_url').closest('.form-group').hide();
+          $('#id_dynamicdns-disable_SSL_cert_check').closest('.form-group').hide();
+          $('#id_dynamicdns-use_http_basic_auth').closest('.form-group').hide();
+          $('#id_dynamicdns-dynamicdns_server').closest('.form-group').show();
+        }else{
+          $('#id_dynamicdns-dynamicdns_update_url').closest('.form-group').show();
+          $('#id_dynamicdns-disable_SSL_cert_check').closest('.form-group').show();
+          $('#id_dynamicdns-use_http_basic_auth').closest('.form-group').show();
+          $('#id_dynamicdns-dynamicdns_server').closest('.form-group').hide();
+          if ($("#id_dynamicdns-service_type option:selected").text() == "noip.com") {
+            $('#id_dynamicdns-dynamicdns_update_url').val(NOIP);
+            $('#id_dynamicdns-use_http_basic_auth').prop('checked', true);
+          }else{
+            $('#id_dynamicdns-use_http_basic_auth').prop('checked', false);
+          }
+          if ($("#id_dynamicdns-service_type option:selected").text() == "selfhost.bz") {
+            $('#id_dynamicdns-dynamicdns_update_url').val(SELFHOST);
+          }
+          if ($("#id_dynamicdns-service_type option:selected").text() == "freedns.afraid.org") {
+              $('#id_dynamicdns-dynamicdns_update_url').val(FREEDNS);
+            }
+          if ($("#id_dynamicdns-service_type option:selected").text() == "other update URL") {
+            $('#id_dynamicdns-dynamicdns_update_url').val('');
+          }
+        }
+      }
+
+      function show_all()
+      {
+          $('#id_dynamicdns-enabled').closest('.form-group').show();
+          $('#id_dynamicdns-service_type').closest('.form-group').show();
+          $('#id_dynamicdns-dynamicdns_server').closest('.form-group').show();
+          $('#id_dynamicdns-dynamicdns_update_url').closest('.form-group').show();
+          $('#id_dynamicdns-disable_SSL_cert_check').closest('.form-group').show();
+          $('#id_dynamicdns-use_http_basic_auth').closest('.form-group').show();
+          $('#id_dynamicdns-dynamicdns_domain').closest('.form-group').show();
+          $('#id_dynamicdns-dynamicdns_user').closest('.form-group').show();
+          $('#id_dynamicdns-dynamicdns_secret').closest('.form-group').show();
+          $('#id_dynamicdns-showpw').closest('.form-group').show();
+          $('#id_dynamicdns-dynamicdns_ipurl').closest('.form-group').show();
```

Comment:
You could assign an additional class to the configuration form for clarity, say `dynamicdns-configuration` and try using `$('dynamicdns-configuration .form-group').show()`